ImageNet/train/dawn_resnet50_tpu_tensorflow17.json

```diff
@@ -0,0 +1,12 @@
+{
+    "version": "v1.0",
+    "author": "Google Cloud TPU",
+    "authorEmail": "cloud-tpu-dawnbench@googlegroups.com",
+    "framework": "TensorFlow v1.7rc1",
+    "codeURL": "https://github.com/tensorflow/tpu/tree/b189540102d6b5b40b1730d7e5ad5c884bae323c/models/experimental/resnet_bfloat16",
+    "model": "ResNet50",
+    "hardware": "GCP N1-standard-2, Cloud TPU",
+    "costPerHour": 6.595,
+    "timestamp": "2017-03-25",
+    "misc": {}
```

Comment:
Please add the values you used for hyperparameters under misc: 

From your codeURL ([resnet_main.py](https://github.com/tensorflow/tpu/blob/b189540102d6b5b40b1730d7e5ad5c884bae323c/models/experimental/resnet_bfloat16/resnet_main.py#L155)) and assuming a batch size of 256, it should be something like:

```
"misc": {
  "optimizer": "SGD with Momentum",
  "momentum": 0.9,
  "weightDecay": 0.0001,
  "batchSize": 256,
  "schedule": [
      {"learning_rate": 0.02, "epochs": 1},
      {"learning_rate": 0.04, "epochs": 1},
      {"learning_rate": 0.06, "epochs": 1},
      {"learning_rate": 0.08, "epochs": 1},
      {"learning_rate": 0.1, "epochs":  26},
      {"learning_rate": 0.01, "epochs":  30},
      {"learning_rate": 0.001, "epochs": 20}, 
      {"learning_rate": 0.0001, "epochs": 15}
    ]
  }

```

Please update with the values you used. If you used a larger batch size, please scale the learning rates accordingly ([L180](https://github.com/tensorflow/tpu/blob/b189540102d6b5b40b1730d7e5ad5c884bae323c/models/experimental/resnet_bfloat16/resnet_main.py#L180)).

Reply:
Updated. The default parameters on the code base should match these.